kahuna/public/js/components/gr-panel-button/gr-panel-button-small.html

```diff
@@ -0,0 +1,29 @@
+<div class="fill-height flex-container"
+     ng:class="{ 'flex-container--rtl': ctrl.position == 'right' }">
+    <button class="button-ico-square clickable" type="button"
+            gr:tooltip="Hide {{ctrl.name}} panel"
+            ng:if="!ctrl.state.hidden" ng:click="ctrl.hidePanel()"
```

Comment:
Can we stick to one attribute per line? otherwise it's easy to miss and harder to read.